config/sync/field.storage.node.su_basic_page_tags_filters.yml

```diff
@@ -0,0 +1,20 @@
+uuid: c5e30d01-74e0-41d5-b487-70f85c72bb05
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - taxonomy
+id: node.su_basic_page_tags_filters
+field_name: su_basic_page_tags_filters
```

Comment:
Can we call this something more similar to what we have elsewhere?

su_page_tags